apps/example-todo-app/tests/api/todo/form-add.test.ts

```diff
@@ -25,27 +24,33 @@ test("POST /todo/form-add", async (t) => {
   t.is(successfulRes.status, 200)
 })
 
-test("Valid formData object passes validation", (t) => {
+test("Valid formData object passes validation and returns successful response", async (t) => {
+  const { axios } = await getTestServer(t)
+
+  axios.defaults.headers.common.Authorization = `Bearer auth_token`
+
+  const bodyFormData = new URLSearchParams()
```

Comment:
```suggestion
  const bodyFormData = new FormData()
```

Reply:
Looks like FormData isn't supported. Seve created an issue to implement support for that: https://github.com/seamapi/nextlove/issues/127